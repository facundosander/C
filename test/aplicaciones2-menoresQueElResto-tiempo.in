menoresQueElRestoTiempo 300000 2
Fin