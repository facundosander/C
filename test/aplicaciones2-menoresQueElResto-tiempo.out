1>Jugador 1: Juan, Uruguay, 20 aÃ±os
Bien.
2>Fin.
